paper/src/main/java/co/aikar/commands/PaperAsyncTabCompleteHandler.java

```diff
@@ -37,7 +37,8 @@
 
     PaperAsyncTabCompleteHandler(PaperCommandManager manager) {
         this.manager = manager;
-        manager.log(LogLevel.INFO, "Enabled Asynchronous Tab Completion Support!");
+        if (!Boolean.getBoolean("aikar.commands.disable-async-message"))
+            manager.log(LogLevel.INFO, "Enabled Asynchronous Tab Completion Support!");
```

Comment:
Please remove this change and then it looks good.

Reply:
should be done.